boards/stm32f3discovery/doc.txt

```diff
@@ -107,6 +108,7 @@ An 3-axis gyroscope is soldered on the board.
 | INT1       | PE0 (IN)      |
 | INT2/DRDY  | PE1 (IN)      |
 
+[1] L3GD20 (up to rev. D01), IG34250D (from rev. E02)
```

Comment:
should have a note here that `Makefile.dep` needs to be adapted for this - or does the driver support run-time detection now?

Reply:
The driver detects the right sensor type for all sensor types that are enabled by pseudomodules. In case of the STM32F3Discovery board, `l3gd20` and `i3g4250d` are enabled so that the driver can detect them.

Reply:
I added a short statement to the footnote.